hack/get_aihub_manifests.sh

```
@@ -0,0 +1,145 @@
+        fi
+    fi
+    return 1
+}
+
+function git_fetch_ref()
+{
+    local repo=$1
+    local ref=$2
+    local dir=$3
+
+    mkdir -p $dir
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)